src/s1reader/s1_burst_slc.py

```diff
@@ -8,7 +8,7 @@
 import isce3
 import numpy as np
 from osgeo import gdal
-
+from scipy.interpolate import InterpolatedUnivariateSpline
```

Comment:
Good point. The reason that I've used `InterpolatedUnivariateSpline` is because there is no guarantee that the grids (azimuth time in this case) are equally spaced. I'll do more search on this. For now, I'll add scipy into `requirements.txt` to ensure that we have everything we need.